.plop/templates/component/package.json.hbs

```diff
@@ -27,6 +27,7 @@
   "source": "src/index.ts",
   "scripts": {
     "build": "vite build -c ../../vite.config.ts && tsc --project tsconfig.build.json",
```

Comment:
@chasedarkly I almost added `concurrently` to `devDependencies` in all the package-level `package.json` files (because it's required for the new `build:watch` commands in each package), but based on how we treat `eslint`, `vite`, `tsc`, etc. it seemed that the convention is to let those tools be managed at the top-level.

Does that sound right to you?

Reply:
Sounds right to me 👍